src/client/common.ts

```diff
@@ -126,50 +126,10 @@ export const serializeDataIfNeeded = function (value: any, requestOptions: any,
         ? configuration.isJsonMime(requestOptions.headers['Content-Type'])
         : nonString;
     return needsSerialization
-        ? JSON.stringify(value !== undefined ? convertMapsAndSetsToPlain(value) : {})
+        ? JSON.stringify(value !== undefined ? value : {})
         : (value || "");
 }
 
-function convertMapsAndSetsToPlain(value: any): any {
```

Comment:
@marcin-cb the client generator does not remove this. we need to keep `convertMapsAndSetsToPlain` function

Reply:
Hmm yeah this looks to be the same issue from my previous PR. I gotta see what's different about my env that generates this incorrectly.

Reply:
Alright replaced it with a correct version.